### DISCUSSION TITLE
upgrade from v5 to v6

### DISCUSSION THREAD

**Discussion by andim27** [nylo-core/nylo#175] [General] [answered]
Please write how to upgrade project from v5 version to v6
Please do not put here upgrade link because so many new features presents in v6
Write short doc here(what was added. in v6, which new  ability in core was added etc....)

**Comment by andim27** [comment]
In v5 was method:   
1) /// Use boot if you need to load data before the view is rendered.
@override
  boot() async {
  }
Where is "boot" method in v6?
2) @override
  Widget loading(BuildContext context) {}
Where is loading in v6?
3) The name 'Session' is defined in the libraries 'package:gotrue/src/types/session.dart' and 'package:nylo_support/helpers/session.dart (via package:nylo_framework/nylo_framework.dart)'.
Why not use: NySession?

Please post comment how to upgrade in this case

**Reply by agordn52** [reply, marked as answer]
I've added a bit more info on the upgrade guide page to explain this. Here's a link https://nylo.dev/docs/6.x/upgrade-guide
